2016-12-16 00:06:14.655 [error] <0.148.0> Supervisor {<0.148.0>,poolboy_sup} had child boss_session_controller started with boss_session_controller:start_link([{name,{local,boss_session_pool}},{worker_module,boss_session_controller},{size,20},{max_overflow,...},...]) at undefined exit with reason shutdown in context shutdown_error
2016-12-16 00:06:14.655 [error] <0.148.0> Supervisor {<0.148.0>,poolboy_sup} had child boss_session_controller started with boss_session_controller:start_link([{name,{local,boss_session_pool}},{worker_module,boss_session_controller},{size,20},{max_overflow,...},...]) at undefined exit with reason noproc in context shutdown_error
2016-12-16 00:06:40.866 [error] <0.294.0> gen_server <0.294.0> terminated with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:route/2 line 172
2016-12-16 00:06:40.866 [error] <0.294.0> CRASH REPORT Process <0.294.0> with 0 neighbours exited with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:route/2 line 172 in gen_server:terminate/7 line 804
2016-12-16 00:06:40.866 [error] <0.293.0> Supervisor {<0.293.0>,boss_router_sup} had child router_controller started with boss_router_controller:start_link([{application,currencyfair_server},{controllers,[]}]) at <0.294.0> exit with reason bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:route/2 line 172 in context child_terminated
2016-12-16 00:06:40.872 [error] <0.301.0>@boss_web_controller_handle_request:handle_request:30 Unhandled Error: exit:{{badarg,
          [{erlang,list_to_atom,[undefined],[]},
           {boss_router_controller,route,2,
               [{file,"src/boss/boss_router_controller.erl"},{line,172}]},
           {boss_router_controller,handle_call,3,
               [{file,"src/boss/boss_router_controller.erl"},{line,64}]},
           {gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,607}]},
           {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,639}]},
           {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},
      {gen_server,call,[<0.294.0>,{route,"/"}]}} Stacktrace: [{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{boss_web_controller_handle_request,process_dynamic_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,245}]},{boss_web_controller_handle_request,process_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,240}]},{timer,tc,3,[{file,"timer.erl"},{line,194}]},{boss_web_controller_handle_request,build_dynamic_response,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,129}]},{boss_web_controller_handle_request,handle_request,4,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,26}]},{mochicow_upgrade,upgrade,4,[{file,"src/mochicow_upgrade.erl"},{line,72}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,442}]}]
2016-12-16 00:06:44.347 [error] <0.302.0> gen_server <0.302.0> terminated with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146
2016-12-16 00:06:44.347 [error] <0.302.0> CRASH REPORT Process <0.302.0> with 0 neighbours exited with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in gen_server:terminate/7 line 804
2016-12-16 00:06:44.347 [error] <0.293.0> Supervisor {<0.293.0>,boss_router_sup} had child router_controller started with boss_router_controller:start_link([{application,currencyfair_server},{controllers,[]}]) at <0.302.0> exit with reason bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in context child_terminated
2016-12-16 00:06:44.347 [error] <0.304.0>@boss_web_controller_handle_request:handle_request:30 Unhandled Error: exit:{{badarg,
          [{erlang,list_to_atom,[undefined],[]},
           {boss_router_controller,handle,2,
               [{file,"src/boss/boss_router_controller.erl"},{line,146}]},
           {boss_router_controller,handle_call,3,
               [{file,"src/boss/boss_router_controller.erl"},{line,59}]},
           {gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,607}]},
           {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,639}]},
           {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},
      {gen_server,call,[<0.302.0>,{handle,404}]}} Stacktrace: [{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{boss_web_controller_handle_request,process_not_found,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,277}]},{boss_web_controller_handle_request,process_dynamic_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,265}]},{boss_web_controller_handle_request,process_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,240}]},{timer,tc,3,[{file,"timer.erl"},{line,194}]},{boss_web_controller_handle_request,build_dynamic_response,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,129}]},{boss_web_controller_handle_request,handle_request,4,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,26}]},{mochicow_upgrade,upgrade,4,[{file,"src/mochicow_upgrade.erl"},{line,72}]}]
2016-12-16 00:06:51.378 [error] <0.305.0> gen_server <0.305.0> terminated with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146
2016-12-16 00:06:51.378 [error] <0.305.0> CRASH REPORT Process <0.305.0> with 0 neighbours exited with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in gen_server:terminate/7 line 804
2016-12-16 00:06:51.379 [error] <0.293.0> Supervisor {<0.293.0>,boss_router_sup} had child router_controller started with boss_router_controller:start_link([{application,currencyfair_server},{controllers,[]}]) at <0.305.0> exit with reason bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in context child_terminated
2016-12-16 00:06:51.379 [error] <0.307.0>@boss_web_controller_handle_request:handle_request:30 Unhandled Error: exit:{{badarg,
          [{erlang,list_to_atom,[undefined],[]},
           {boss_router_controller,handle,2,
               [{file,"src/boss/boss_router_controller.erl"},{line,146}]},
           {boss_router_controller,handle_call,3,
               [{file,"src/boss/boss_router_controller.erl"},{line,59}]},
           {gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,607}]},
           {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,639}]},
           {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},
      {gen_server,call,[<0.305.0>,{handle,404}]}} Stacktrace: [{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{boss_web_controller_handle_request,process_not_found,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,277}]},{boss_web_controller_handle_request,process_dynamic_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,265}]},{boss_web_controller_handle_request,process_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,240}]},{timer,tc,3,[{file,"timer.erl"},{line,194}]},{boss_web_controller_handle_request,build_dynamic_response,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,129}]},{boss_web_controller_handle_request,handle_request,4,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,26}]},{mochicow_upgrade,upgrade,4,[{file,"src/mochicow_upgrade.erl"},{line,72}]}]
2016-12-16 00:08:42.224 [error] <0.300.0> gen_server <0.300.0> terminated with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:route/2 line 172
2016-12-16 00:08:42.224 [error] <0.300.0> CRASH REPORT Process <0.300.0> with 0 neighbours exited with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:route/2 line 172 in gen_server:terminate/7 line 804
2016-12-16 00:08:42.224 [error] <0.299.0> Supervisor {<0.299.0>,boss_router_sup} had child router_controller started with boss_router_controller:start_link([{application,currencyfair_server},{controllers,["currencyfair_server_home_controller","currency..."]}]) at <0.300.0> exit with reason bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:route/2 line 172 in context child_terminated
2016-12-16 00:08:42.243 [error] <0.325.0>@boss_web_controller_handle_request:handle_request:30 Unhandled Error: exit:{{badarg,
          [{erlang,list_to_atom,[undefined],[]},
           {boss_router_controller,route,2,
               [{file,"src/boss/boss_router_controller.erl"},{line,172}]},
           {boss_router_controller,handle_call,3,
               [{file,"src/boss/boss_router_controller.erl"},{line,64}]},
           {gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,607}]},
           {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,639}]},
           {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},
      {gen_server,call,[<0.300.0>,{route,"/"}]}} Stacktrace: [{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{boss_web_controller_handle_request,process_dynamic_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,245}]},{boss_web_controller_handle_request,process_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,236}]},{timer,tc,3,[{file,"timer.erl"},{line,194}]},{boss_web_controller_handle_request,build_dynamic_response,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,129}]},{boss_web_controller_handle_request,handle_request,4,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,26}]},{mochicow_upgrade,upgrade,4,[{file,"src/mochicow_upgrade.erl"},{line,72}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,442}]}]
2016-12-16 00:13:03.125 [error] <0.327.0> gen_server <0.327.0> terminated with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:route/2 line 172
2016-12-16 00:13:03.127 [error] <0.327.0> CRASH REPORT Process <0.327.0> with 0 neighbours exited with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:route/2 line 172 in gen_server:terminate/7 line 804
2016-12-16 00:13:03.129 [error] <0.299.0> Supervisor {<0.299.0>,boss_router_sup} had child router_controller started with boss_router_controller:start_link([{application,currencyfair_server},{controllers,["currencyfair_server_home_controller","currency..."]}]) at <0.327.0> exit with reason bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:route/2 line 172 in context child_terminated
2016-12-16 00:13:03.131 [error] <0.338.0>@boss_web_controller_handle_request:handle_request:30 Unhandled Error: exit:{{badarg,
          [{erlang,list_to_atom,[undefined],[]},
           {boss_router_controller,route,2,
               [{file,"src/boss/boss_router_controller.erl"},{line,172}]},
           {boss_router_controller,handle_call,3,
               [{file,"src/boss/boss_router_controller.erl"},{line,64}]},
           {gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,607}]},
           {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,639}]},
           {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},
      {gen_server,call,[<0.327.0>,{route,"/"}]}} Stacktrace: [{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{boss_web_controller_handle_request,process_dynamic_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,245}]},{boss_web_controller_handle_request,process_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,236}]},{timer,tc,3,[{file,"timer.erl"},{line,194}]},{boss_web_controller_handle_request,build_dynamic_response,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,129}]},{boss_web_controller_handle_request,handle_request,4,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,26}]},{mochicow_upgrade,upgrade,4,[{file,"src/mochicow_upgrade.erl"},{line,72}]},{cowboy_protocol,execute,4,[{file,"src/cowboy_protocol.erl"},{line,442}]}]
2016-12-16 00:14:15.161 [error] <0.340.0> gen_server <0.340.0> terminated with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146
2016-12-16 00:14:15.162 [error] <0.340.0> CRASH REPORT Process <0.340.0> with 0 neighbours exited with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in gen_server:terminate/7 line 804
2016-12-16 00:14:15.162 [error] <0.299.0> Supervisor {<0.299.0>,boss_router_sup} had child router_controller started with boss_router_controller:start_link([{application,currencyfair_server},{controllers,["currencyfair_server_home_controller","currency..."]}]) at <0.340.0> exit with reason bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in context child_terminated
2016-12-16 00:14:15.163 [error] <0.343.0>@boss_web_controller_handle_request:handle_request:30 Unhandled Error: exit:{{badarg,
          [{erlang,list_to_atom,[undefined],[]},
           {boss_router_controller,handle,2,
               [{file,"src/boss/boss_router_controller.erl"},{line,146}]},
           {boss_router_controller,handle_call,3,
               [{file,"src/boss/boss_router_controller.erl"},{line,59}]},
           {gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,607}]},
           {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,639}]},
           {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},
      {gen_server,call,[<0.340.0>,{handle,404}]}} Stacktrace: [{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{boss_web_controller_handle_request,process_not_found,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,277}]},{boss_web_controller_handle_request,process_dynamic_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,265}]},{boss_web_controller_handle_request,process_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,236}]},{timer,tc,3,[{file,"timer.erl"},{line,194}]},{boss_web_controller_handle_request,build_dynamic_response,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,129}]},{boss_web_controller_handle_request,handle_request,4,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,26}]},{mochicow_upgrade,upgrade,4,[{file,"src/mochicow_upgrade.erl"},{line,72}]}]
2016-12-16 00:14:31.341 [error] <0.345.0> gen_server <0.345.0> terminated with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146
2016-12-16 00:14:31.341 [error] <0.345.0> CRASH REPORT Process <0.345.0> with 0 neighbours exited with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in gen_server:terminate/7 line 804
2016-12-16 00:14:31.341 [error] <0.299.0> Supervisor {<0.299.0>,boss_router_sup} had child router_controller started with boss_router_controller:start_link([{application,currencyfair_server},{controllers,["currencyfair_server_home_controller","currency..."]}]) at <0.345.0> exit with reason bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in context child_terminated
2016-12-16 00:14:31.342 [error] <0.347.0>@boss_web_controller_handle_request:handle_request:30 Unhandled Error: exit:{{badarg,
          [{erlang,list_to_atom,[undefined],[]},
           {boss_router_controller,handle,2,
               [{file,"src/boss/boss_router_controller.erl"},{line,146}]},
           {boss_router_controller,handle_call,3,
               [{file,"src/boss/boss_router_controller.erl"},{line,59}]},
           {gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,607}]},
           {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,639}]},
           {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},
      {gen_server,call,[<0.345.0>,{handle,404}]}} Stacktrace: [{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{boss_web_controller_handle_request,process_not_found,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,277}]},{boss_web_controller_handle_request,process_dynamic_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,265}]},{boss_web_controller_handle_request,process_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,236}]},{timer,tc,3,[{file,"timer.erl"},{line,194}]},{boss_web_controller_handle_request,build_dynamic_response,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,129}]},{boss_web_controller_handle_request,handle_request,4,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,26}]},{mochicow_upgrade,upgrade,4,[{file,"src/mochicow_upgrade.erl"},{line,72}]}]
2016-12-16 00:14:37.242 [error] <0.148.0> Supervisor {<0.148.0>,poolboy_sup} had child boss_session_controller started with boss_session_controller:start_link([{name,{local,boss_session_pool}},{worker_module,boss_session_controller},{size,20},{max_overflow,...},...]) at undefined exit with reason shutdown in context shutdown_error
2016-12-16 00:16:50.830 [error] <0.304.0> gen_server <0.304.0> terminated with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146
2016-12-16 00:16:50.831 [error] <0.304.0> CRASH REPORT Process <0.304.0> with 0 neighbours exited with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in gen_server:terminate/7 line 804
2016-12-16 00:16:50.831 [error] <0.303.0> Supervisor {<0.303.0>,boss_router_sup} had child router_controller started with boss_router_controller:start_link([{application,currencyfair_server},{controllers,["currencyfair_server_home_controller","currency..."]}]) at <0.304.0> exit with reason bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in context child_terminated
2016-12-16 00:16:50.839 [error] <0.326.0>@boss_web_controller_handle_request:handle_request:30 Unhandled Error: exit:{{badarg,
          [{erlang,list_to_atom,[undefined],[]},
           {boss_router_controller,handle,2,
               [{file,"src/boss/boss_router_controller.erl"},{line,146}]},
           {boss_router_controller,handle_call,3,
               [{file,"src/boss/boss_router_controller.erl"},{line,59}]},
           {gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,607}]},
           {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,639}]},
           {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},
      {gen_server,call,[<0.304.0>,{handle,404}]}} Stacktrace: [{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{boss_web_controller_handle_request,process_not_found,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,277}]},{boss_web_controller_handle_request,process_dynamic_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,265}]},{boss_web_controller_handle_request,process_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,236}]},{timer,tc,3,[{file,"timer.erl"},{line,194}]},{boss_web_controller_handle_request,build_dynamic_response,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,129}]},{boss_web_controller_handle_request,handle_request,4,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,26}]},{mochicow_upgrade,upgrade,4,[{file,"src/mochicow_upgrade.erl"},{line,72}]}]
2016-12-16 00:16:54.785 [error] <0.328.0> gen_server <0.328.0> terminated with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146
2016-12-16 00:16:54.786 [error] <0.328.0> CRASH REPORT Process <0.328.0> with 0 neighbours exited with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in gen_server:terminate/7 line 804
2016-12-16 00:16:54.786 [error] <0.303.0> Supervisor {<0.303.0>,boss_router_sup} had child router_controller started with boss_router_controller:start_link([{application,currencyfair_server},{controllers,["currencyfair_server_home_controller","currency..."]}]) at <0.328.0> exit with reason bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in context child_terminated
2016-12-16 00:16:54.787 [error] <0.330.0>@boss_web_controller_handle_request:handle_request:30 Unhandled Error: exit:{{badarg,
          [{erlang,list_to_atom,[undefined],[]},
           {boss_router_controller,handle,2,
               [{file,"src/boss/boss_router_controller.erl"},{line,146}]},
           {boss_router_controller,handle_call,3,
               [{file,"src/boss/boss_router_controller.erl"},{line,59}]},
           {gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,607}]},
           {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,639}]},
           {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},
      {gen_server,call,[<0.328.0>,{handle,404}]}} Stacktrace: [{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{boss_web_controller_handle_request,process_not_found,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,277}]},{boss_web_controller_handle_request,process_dynamic_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,265}]},{boss_web_controller_handle_request,process_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,236}]},{timer,tc,3,[{file,"timer.erl"},{line,194}]},{boss_web_controller_handle_request,build_dynamic_response,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,129}]},{boss_web_controller_handle_request,handle_request,4,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,26}]},{mochicow_upgrade,upgrade,4,[{file,"src/mochicow_upgrade.erl"},{line,72}]}]
2016-12-16 00:17:05.004 [error] <0.332.0> gen_server <0.332.0> terminated with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146
2016-12-16 00:17:05.005 [error] <0.332.0> CRASH REPORT Process <0.332.0> with 0 neighbours exited with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in gen_server:terminate/7 line 804
2016-12-16 00:17:05.005 [error] <0.303.0> Supervisor {<0.303.0>,boss_router_sup} had child router_controller started with boss_router_controller:start_link([{application,currencyfair_server},{controllers,["currencyfair_server_home_controller","currency..."]}]) at <0.332.0> exit with reason bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in context child_terminated
2016-12-16 00:17:05.006 [error] <0.336.0>@boss_web_controller_handle_request:handle_request:30 Unhandled Error: exit:{{badarg,
          [{erlang,list_to_atom,[undefined],[]},
           {boss_router_controller,handle,2,
               [{file,"src/boss/boss_router_controller.erl"},{line,146}]},
           {boss_router_controller,handle_call,3,
               [{file,"src/boss/boss_router_controller.erl"},{line,59}]},
           {gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,607}]},
           {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,639}]},
           {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},
      {gen_server,call,[<0.332.0>,{handle,404}]}} Stacktrace: [{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{boss_web_controller_handle_request,process_not_found,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,277}]},{boss_web_controller_handle_request,process_dynamic_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,256}]},{boss_web_controller_handle_request,process_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,236}]},{timer,tc,3,[{file,"timer.erl"},{line,194}]},{boss_web_controller_handle_request,build_dynamic_response,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,129}]},{boss_web_controller_handle_request,handle_request,4,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,26}]},{mochicow_upgrade,upgrade,4,[{file,"src/mochicow_upgrade.erl"},{line,72}]}]
2016-12-16 00:17:06.279 [error] <0.338.0> gen_server <0.338.0> terminated with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146
2016-12-16 00:17:06.279 [error] <0.338.0> CRASH REPORT Process <0.338.0> with 0 neighbours exited with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in gen_server:terminate/7 line 804
2016-12-16 00:17:06.280 [error] <0.303.0> Supervisor {<0.303.0>,boss_router_sup} had child router_controller started with boss_router_controller:start_link([{application,currencyfair_server},{controllers,["currencyfair_server_home_controller","currency..."]}]) at <0.338.0> exit with reason bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in context child_terminated
2016-12-16 00:17:06.281 [error] <0.340.0>@boss_web_controller_handle_request:handle_request:30 Unhandled Error: exit:{{badarg,
          [{erlang,list_to_atom,[undefined],[]},
           {boss_router_controller,handle,2,
               [{file,"src/boss/boss_router_controller.erl"},{line,146}]},
           {boss_router_controller,handle_call,3,
               [{file,"src/boss/boss_router_controller.erl"},{line,59}]},
           {gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,607}]},
           {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,639}]},
           {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},
      {gen_server,call,[<0.338.0>,{handle,404}]}} Stacktrace: [{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{boss_web_controller_handle_request,process_not_found,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,277}]},{boss_web_controller_handle_request,process_dynamic_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,256}]},{boss_web_controller_handle_request,process_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,236}]},{timer,tc,3,[{file,"timer.erl"},{line,194}]},{boss_web_controller_handle_request,build_dynamic_response,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,129}]},{boss_web_controller_handle_request,handle_request,4,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,26}]},{mochicow_upgrade,upgrade,4,[{file,"src/mochicow_upgrade.erl"},{line,72}]}]
2016-12-16 00:17:07.514 [error] <0.342.0> gen_server <0.342.0> terminated with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146
2016-12-16 00:17:07.514 [error] <0.342.0> CRASH REPORT Process <0.342.0> with 0 neighbours exited with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in gen_server:terminate/7 line 804
2016-12-16 00:17:07.514 [error] <0.303.0> Supervisor {<0.303.0>,boss_router_sup} had child router_controller started with boss_router_controller:start_link([{application,currencyfair_server},{controllers,["currencyfair_server_home_controller","currency..."]}]) at <0.342.0> exit with reason bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in context child_terminated
2016-12-16 00:17:07.515 [error] <0.344.0>@boss_web_controller_handle_request:handle_request:30 Unhandled Error: exit:{{badarg,
          [{erlang,list_to_atom,[undefined],[]},
           {boss_router_controller,handle,2,
               [{file,"src/boss/boss_router_controller.erl"},{line,146}]},
           {boss_router_controller,handle_call,3,
               [{file,"src/boss/boss_router_controller.erl"},{line,59}]},
           {gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,607}]},
           {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,639}]},
           {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},
      {gen_server,call,[<0.342.0>,{handle,404}]}} Stacktrace: [{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{boss_web_controller_handle_request,process_not_found,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,277}]},{boss_web_controller_handle_request,process_dynamic_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,256}]},{boss_web_controller_handle_request,process_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,236}]},{timer,tc,3,[{file,"timer.erl"},{line,194}]},{boss_web_controller_handle_request,build_dynamic_response,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,129}]},{boss_web_controller_handle_request,handle_request,4,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,26}]},{mochicow_upgrade,upgrade,4,[{file,"src/mochicow_upgrade.erl"},{line,72}]}]
2016-12-16 00:19:42.109 [error] <0.346.0> gen_server <0.346.0> terminated with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146
2016-12-16 00:19:42.110 [error] <0.346.0> CRASH REPORT Process <0.346.0> with 0 neighbours exited with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in gen_server:terminate/7 line 804
2016-12-16 00:19:42.110 [error] <0.303.0> Supervisor {<0.303.0>,boss_router_sup} had child router_controller started with boss_router_controller:start_link([{application,currencyfair_server},{controllers,["currencyfair_server_home_controller","currency..."]}]) at <0.346.0> exit with reason bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in context child_terminated
2016-12-16 00:19:42.111 [error] <0.349.0>@boss_web_controller_handle_request:handle_request:30 Unhandled Error: exit:{{badarg,
          [{erlang,list_to_atom,[undefined],[]},
           {boss_router_controller,handle,2,
               [{file,"src/boss/boss_router_controller.erl"},{line,146}]},
           {boss_router_controller,handle_call,3,
               [{file,"src/boss/boss_router_controller.erl"},{line,59}]},
           {gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,607}]},
           {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,639}]},
           {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},
      {gen_server,call,[<0.346.0>,{handle,404}]}} Stacktrace: [{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{boss_web_controller_handle_request,process_not_found,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,277}]},{boss_web_controller_handle_request,process_dynamic_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,256}]},{boss_web_controller_handle_request,process_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,236}]},{timer,tc,3,[{file,"timer.erl"},{line,194}]},{boss_web_controller_handle_request,build_dynamic_response,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,129}]},{boss_web_controller_handle_request,handle_request,4,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,26}]},{mochicow_upgrade,upgrade,4,[{file,"src/mochicow_upgrade.erl"},{line,72}]}]
2016-12-16 00:19:44.543 [error] <0.353.0> gen_server <0.353.0> terminated with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146
2016-12-16 00:19:44.544 [error] <0.353.0> CRASH REPORT Process <0.353.0> with 0 neighbours exited with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in gen_server:terminate/7 line 804
2016-12-16 00:19:44.544 [error] <0.303.0> Supervisor {<0.303.0>,boss_router_sup} had child router_controller started with boss_router_controller:start_link([{application,currencyfair_server},{controllers,["currencyfair_server_home_controller","currency..."]}]) at <0.353.0> exit with reason bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in context child_terminated
2016-12-16 00:19:44.544 [error] <0.355.0>@boss_web_controller_handle_request:handle_request:30 Unhandled Error: exit:{{badarg,
          [{erlang,list_to_atom,[undefined],[]},
           {boss_router_controller,handle,2,
               [{file,"src/boss/boss_router_controller.erl"},{line,146}]},
           {boss_router_controller,handle_call,3,
               [{file,"src/boss/boss_router_controller.erl"},{line,59}]},
           {gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,607}]},
           {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,639}]},
           {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},
      {gen_server,call,[<0.353.0>,{handle,404}]}} Stacktrace: [{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{boss_web_controller_handle_request,process_not_found,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,277}]},{boss_web_controller_handle_request,process_dynamic_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,265}]},{boss_web_controller_handle_request,process_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,236}]},{timer,tc,3,[{file,"timer.erl"},{line,194}]},{boss_web_controller_handle_request,build_dynamic_response,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,129}]},{boss_web_controller_handle_request,handle_request,4,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,26}]},{mochicow_upgrade,upgrade,4,[{file,"src/mochicow_upgrade.erl"},{line,72}]}]
2016-12-16 00:19:45.336 [error] <0.357.0> gen_server <0.357.0> terminated with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146
2016-12-16 00:19:45.337 [error] <0.357.0> CRASH REPORT Process <0.357.0> with 0 neighbours exited with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in gen_server:terminate/7 line 804
2016-12-16 00:19:45.337 [error] <0.359.0>@boss_web_controller_handle_request:handle_request:30 Unhandled Error: exit:{{badarg,
          [{erlang,list_to_atom,[undefined],[]},
           {boss_router_controller,handle,2,
               [{file,"src/boss/boss_router_controller.erl"},{line,146}]},
           {boss_router_controller,handle_call,3,
               [{file,"src/boss/boss_router_controller.erl"},{line,59}]},
           {gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,607}]},
           {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,639}]},
           {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},
      {gen_server,call,[<0.357.0>,{handle,404}]}} Stacktrace: [{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{boss_web_controller_handle_request,process_not_found,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,277}]},{boss_web_controller_handle_request,process_dynamic_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,265}]},{boss_web_controller_handle_request,process_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,236}]},{timer,tc,3,[{file,"timer.erl"},{line,194}]},{boss_web_controller_handle_request,build_dynamic_response,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,129}]},{boss_web_controller_handle_request,handle_request,4,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,26}]},{mochicow_upgrade,upgrade,4,[{file,"src/mochicow_upgrade.erl"},{line,72}]}]
2016-12-16 00:19:45.337 [error] <0.303.0> Supervisor {<0.303.0>,boss_router_sup} had child router_controller started with boss_router_controller:start_link([{application,currencyfair_server},{controllers,["currencyfair_server_home_controller","currency..."]}]) at <0.357.0> exit with reason bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in context child_terminated
2016-12-16 00:20:01.462 [error] <0.361.0> gen_server <0.361.0> terminated with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146
2016-12-16 00:20:01.463 [error] <0.361.0> CRASH REPORT Process <0.361.0> with 0 neighbours exited with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in gen_server:terminate/7 line 804
2016-12-16 00:20:01.464 [error] <0.303.0> Supervisor {<0.303.0>,boss_router_sup} had child router_controller started with boss_router_controller:start_link([{application,currencyfair_server},{controllers,["currencyfair_server_home_controller","currency..."]}]) at <0.361.0> exit with reason bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in context child_terminated
2016-12-16 00:20:01.464 [error] <0.363.0>@boss_web_controller_handle_request:handle_request:30 Unhandled Error: exit:{{badarg,
          [{erlang,list_to_atom,[undefined],[]},
           {boss_router_controller,handle,2,
               [{file,"src/boss/boss_router_controller.erl"},{line,146}]},
           {boss_router_controller,handle_call,3,
               [{file,"src/boss/boss_router_controller.erl"},{line,59}]},
           {gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,607}]},
           {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,639}]},
           {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},
      {gen_server,call,[<0.361.0>,{handle,404}]}} Stacktrace: [{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{boss_web_controller_handle_request,process_not_found,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,277}]},{boss_web_controller_handle_request,process_dynamic_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,265}]},{boss_web_controller_handle_request,process_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,236}]},{timer,tc,3,[{file,"timer.erl"},{line,194}]},{boss_web_controller_handle_request,build_dynamic_response,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,129}]},{boss_web_controller_handle_request,handle_request,4,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,26}]},{mochicow_upgrade,upgrade,4,[{file,"src/mochicow_upgrade.erl"},{line,72}]}]
2016-12-16 00:45:45.445 [error] <0.371.0>@boss_web_controller:call_controller_action:319 Error in controller error:badarg Stacktrace: [{io,format,[<0.190.0>,"In validation, ~p",[]],[]},{transaction,validation_tests,1,[{file,"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/model/transaction.erl"},{line,7}]},{boss_db,validate_record,1,[{file,"src/boss_db.erl"},{line,451}]},{boss_db,save_record,2,[{file,"src/boss_db.erl"},{line,398}]},{currencyfair_server_transactions_controller,record,3,[{file,"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/controller/currencyfair_server_transactions_controller.erl"},{line,32}]},{boss_web_controller,call_controller_action,3,[{file,"src/boss/boss_web_controller.erl"},{line,316}]},{boss_web_controller,apply_action,4,[{file,"src/boss/boss_web_controller.erl"},{line,308}]},{boss_web_controller,execute_action_inner,9,[{file,"src/boss/boss_web_controller.erl"},{line,287}]}]
2016-12-16 00:45:45.449 [error] <0.371.0> Error in controller, see console.log for details

2016-12-16 00:45:45.450 [error] <0.365.0> gen_server <0.365.0> terminated with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146
2016-12-16 00:45:45.450 [error] <0.365.0> CRASH REPORT Process <0.365.0> with 0 neighbours exited with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in gen_server:terminate/7 line 804
2016-12-16 00:45:45.451 [error] <0.303.0> Supervisor {<0.303.0>,boss_router_sup} had child router_controller started with boss_router_controller:start_link([{application,currencyfair_server},{controllers,["currencyfair_server_home_controller","currency..."]}]) at <0.365.0> exit with reason bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in context child_terminated
2016-12-16 00:45:45.452 [error] <0.371.0>@boss_web_controller_handle_request:handle_request:30 Unhandled Error: exit:{{badarg,
          [{erlang,list_to_atom,[undefined],[]},
           {boss_router_controller,handle,2,
               [{file,"src/boss/boss_router_controller.erl"},{line,146}]},
           {boss_router_controller,handle_call,3,
               [{file,"src/boss/boss_router_controller.erl"},{line,59}]},
           {gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,607}]},
           {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,639}]},
           {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},
      {gen_server,call,[<0.365.0>,{handle,500}]}} Stacktrace: [{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{boss_web_controller_handle_request,process_error,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,298}]},{boss_web_controller_handle_request,process_dynamic_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,270}]},{boss_web_controller_handle_request,process_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,236}]},{timer,tc,3,[{file,"timer.erl"},{line,194}]},{boss_web_controller_handle_request,build_dynamic_response,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,129}]},{boss_web_controller_handle_request,handle_request,4,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,26}]},{mochicow_upgrade,upgrade,4,[{file,"src/mochicow_upgrade.erl"},{line,72}]}]
2016-12-16 00:48:20.031 [error] <0.378.0>@boss_web_controller:call_controller_action:319 Error in controller error:badarg Stacktrace: [{io,format,[<0.190.0>,"In validation ~p~n",[]],[]},{transaction,validation_tests,1,[{file,"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/model/transaction.erl"},{line,7}]},{boss_db,validate_record,1,[{file,"src/boss_db.erl"},{line,451}]},{boss_db,save_record,2,[{file,"src/boss_db.erl"},{line,398}]},{currencyfair_server_transactions_controller,record,3,[{file,"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/controller/currencyfair_server_transactions_controller.erl"},{line,32}]},{boss_web_controller,call_controller_action,3,[{file,"src/boss/boss_web_controller.erl"},{line,316}]},{boss_web_controller,apply_action,4,[{file,"src/boss/boss_web_controller.erl"},{line,308}]},{boss_web_controller,execute_action_inner,9,[{file,"src/boss/boss_web_controller.erl"},{line,287}]}]
2016-12-16 00:48:20.033 [error] <0.378.0> Error in controller, see console.log for details

2016-12-16 00:48:20.035 [error] <0.375.0> gen_server <0.375.0> terminated with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146
2016-12-16 00:48:20.035 [error] <0.375.0> CRASH REPORT Process <0.375.0> with 0 neighbours exited with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in gen_server:terminate/7 line 804
2016-12-16 00:48:20.035 [error] <0.303.0> Supervisor {<0.303.0>,boss_router_sup} had child router_controller started with boss_router_controller:start_link([{application,currencyfair_server},{controllers,["currencyfair_server_home_controller","currency..."]}]) at <0.375.0> exit with reason bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in context child_terminated
2016-12-16 00:48:20.036 [error] <0.378.0>@boss_web_controller_handle_request:handle_request:30 Unhandled Error: exit:{{badarg,
          [{erlang,list_to_atom,[undefined],[]},
           {boss_router_controller,handle,2,
               [{file,"src/boss/boss_router_controller.erl"},{line,146}]},
           {boss_router_controller,handle_call,3,
               [{file,"src/boss/boss_router_controller.erl"},{line,59}]},
           {gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,607}]},
           {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,639}]},
           {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},
      {gen_server,call,[<0.375.0>,{handle,500}]}} Stacktrace: [{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{boss_web_controller_handle_request,process_error,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,298}]},{boss_web_controller_handle_request,process_dynamic_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,270}]},{boss_web_controller_handle_request,process_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,236}]},{timer,tc,3,[{file,"timer.erl"},{line,194}]},{boss_web_controller_handle_request,build_dynamic_response,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,129}]},{boss_web_controller_handle_request,handle_request,4,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,26}]},{mochicow_upgrade,upgrade,4,[{file,"src/mochicow_upgrade.erl"},{line,72}]}]
2016-12-16 00:48:28.014 [error] <0.383.0>@boss_web_controller:call_controller_action:319 Error in controller error:badarg Stacktrace: [{io,format,[<0.190.0>,"In validation ~p~n",[]],[]},{transaction,validation_tests,1,[{file,"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/model/transaction.erl"},{line,7}]},{boss_db,validate_record,1,[{file,"src/boss_db.erl"},{line,451}]},{boss_db,save_record,2,[{file,"src/boss_db.erl"},{line,398}]},{currencyfair_server_transactions_controller,record,3,[{file,"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/controller/currencyfair_server_transactions_controller.erl"},{line,32}]},{boss_web_controller,call_controller_action,3,[{file,"src/boss/boss_web_controller.erl"},{line,316}]},{boss_web_controller,apply_action,4,[{file,"src/boss/boss_web_controller.erl"},{line,308}]},{boss_web_controller,execute_action_inner,9,[{file,"src/boss/boss_web_controller.erl"},{line,287}]}]
2016-12-16 00:48:28.021 [error] <0.383.0> Error in controller, see console.log for details

2016-12-16 00:48:28.021 [error] <0.381.0> gen_server <0.381.0> terminated with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146
2016-12-16 00:48:28.022 [error] <0.381.0> CRASH REPORT Process <0.381.0> with 0 neighbours exited with reason: bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in gen_server:terminate/7 line 804
2016-12-16 00:48:28.022 [error] <0.303.0> Supervisor {<0.303.0>,boss_router_sup} had child router_controller started with boss_router_controller:start_link([{application,currencyfair_server},{controllers,["currencyfair_server_home_controller","currency..."]}]) at <0.381.0> exit with reason bad argument in call to erlang:list_to_atom(undefined) in boss_router_controller:handle/2 line 146 in context child_terminated
2016-12-16 00:48:28.023 [error] <0.383.0>@boss_web_controller_handle_request:handle_request:30 Unhandled Error: exit:{{badarg,
          [{erlang,list_to_atom,[undefined],[]},
           {boss_router_controller,handle,2,
               [{file,"src/boss/boss_router_controller.erl"},{line,146}]},
           {boss_router_controller,handle_call,3,
               [{file,"src/boss/boss_router_controller.erl"},{line,59}]},
           {gen_server,try_handle_call,4,[{file,"gen_server.erl"},{line,607}]},
           {gen_server,handle_msg,5,[{file,"gen_server.erl"},{line,639}]},
           {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,237}]}]},
      {gen_server,call,[<0.381.0>,{handle,500}]}} Stacktrace: [{gen_server,call,2,[{file,"gen_server.erl"},{line,182}]},{boss_web_controller_handle_request,process_error,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,298}]},{boss_web_controller_handle_request,process_dynamic_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,270}]},{boss_web_controller_handle_request,process_request,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,236}]},{timer,tc,3,[{file,"timer.erl"},{line,194}]},{boss_web_controller_handle_request,build_dynamic_response,5,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,129}]},{boss_web_controller_handle_request,handle_request,4,[{file,"src/boss/boss_web_controller_handle_request.erl"},{line,26}]},{mochicow_upgrade,upgrade,4,[{file,"src/mochicow_upgrade.erl"},{line,72}]}]
2016-12-16 00:49:04.790 [error] <0.387.0>@boss_web_controller:call_controller_action:319 Error in controller error:badarg Stacktrace: [{io,format,[<0.190.0>,"In validation ~p~n",[]],[]},{transaction,validation_tests,1,[{file,"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/model/transaction.erl"},{line,7}]},{boss_db,validate_record,1,[{file,"src/boss_db.erl"},{line,451}]},{boss_db,save_record,2,[{file,"src/boss_db.erl"},{line,398}]},{currencyfair_server_transactions_controller,record,3,[{file,"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/controller/currencyfair_server_transactions_controller.erl"},{line,32}]},{boss_web_controller,call_controller_action,3,[{file,"src/boss/boss_web_controller.erl"},{line,316}]},{boss_web_controller,apply_action,4,[{file,"src/boss/boss_web_controller.erl"},{line,308}]},{boss_web_controller,execute_action_inner,9,[{file,"src/boss/boss_web_controller.erl"},{line,287}]}]
2016-12-16 00:49:04.791 [error] <0.387.0> Error in controller, see console.log for details

2016-12-16 00:49:04.833 [error] <0.387.0> POST /transactions/record [currencyfair_server] 500 137ms
2016-12-16 00:51:03.052 [error] <0.391.0>@boss_web_controller:call_controller_action:319 Error in controller error:badarg Stacktrace: [{erlang,length,[1000],[]},{transaction,-validation_tests/1-fun-3-,1,[{file,"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/model/transaction.erl"},{line,15}]},{boss_db,-validate_record/1-lc$^0/1-0-,1,[{file,"src/boss_db.erl"},{line,451}]},{boss_db,validate_record,1,[{file,"src/boss_db.erl"},{line,451}]},{boss_db,save_record,2,[{file,"src/boss_db.erl"},{line,398}]},{currencyfair_server_transactions_controller,record,3,[{file,"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/controller/currencyfair_server_transactions_controller.erl"},{line,32}]},{boss_web_controller,call_controller_action,3,[{file,"src/boss/boss_web_controller.erl"},{line,316}]},{boss_web_controller,apply_action,4,[{file,"src/boss/boss_web_controller.erl"},{line,308}]}]
2016-12-16 00:51:03.053 [error] <0.391.0> Error in controller, see console.log for details

2016-12-16 00:51:03.054 [error] <0.391.0> POST /transactions/record [currencyfair_server] 500 141ms
2016-12-16 00:51:14.182 [error] <0.394.0>@boss_web_controller:call_controller_action:319 Error in controller error:badarg Stacktrace: [{erlang,length,[1000],[]},{transaction,-validation_tests/1-fun-3-,1,[{file,"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/model/transaction.erl"},{line,15}]},{boss_db,-validate_record/1-lc$^0/1-0-,1,[{file,"src/boss_db.erl"},{line,451}]},{boss_db,validate_record,1,[{file,"src/boss_db.erl"},{line,451}]},{boss_db,save_record,2,[{file,"src/boss_db.erl"},{line,398}]},{currencyfair_server_transactions_controller,record,3,[{file,"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/controller/currencyfair_server_transactions_controller.erl"},{line,32}]},{boss_web_controller,call_controller_action,3,[{file,"src/boss/boss_web_controller.erl"},{line,316}]},{boss_web_controller,apply_action,4,[{file,"src/boss/boss_web_controller.erl"},{line,308}]}]
2016-12-16 00:51:14.185 [error] <0.394.0> Error in controller, see console.log for details

2016-12-16 00:51:14.186 [error] <0.394.0> POST /transactions/record [currencyfair_server] 500 16ms
2016-12-16 01:08:44.677 [error] <0.434.0>@boss_load:compile_and_accumulate_errors:261 Compile Error, "src/controller/currencyfair_server_transactions_controller.erl" -> [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/controller/currencyfair_server_transactions_controller.erl",[{36,erl_lint,{undefined_function,{bin_to_list,1}}}]}]
2016-12-16 02:55:15.690 [error] <0.152.0> Supervisor {<0.152.0>,poolboy_sup} had child boss_session_controller started with boss_session_controller:start_link([{name,{local,boss_session_pool}},{worker_module,boss_session_controller},{size,20},{max_overflow,...},...]) at undefined exit with reason shutdown in context shutdown_error
2016-12-16 13:02:36.886 [error] <0.94.0>@boss_load:compile_and_accumulate_errors:258 Compile Error, "src/test/functional/server_test.erl" -> [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_test.erl",[{10,erl_parse,["syntax error before: ","']'"]}]}]
2016-12-16 13:02:36.886 [error] <0.94.0>@boss_load:make_all_modules:88 Load Module Error test_modules : [[{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_test.erl",[{10,erl_parse,["syntax error before: ","']'"]}]}]]
2016-12-16 13:17:35.150 [error] <0.148.0> Supervisor {<0.148.0>,poolboy_sup} had child boss_session_controller started with boss_session_controller:start_link([{name,{local,boss_session_pool}},{worker_module,boss_session_controller},{size,20},{max_overflow,...},...]) at undefined exit with reason shutdown in context shutdown_error
2016-12-16 13:17:54.370 [error] <0.2.0>@boss_load:compile_and_accumulate_errors:258 Compile Error, "src/test/functional/server_test.erl" -> [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_test.erl",[{10,erl_parse,["syntax error before: ","']'"]}]}]
2016-12-16 13:17:54.370 [error] <0.2.0>@boss_load:make_all_modules:88 Load Module Error test_modules : [[{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_test.erl",[{10,erl_parse,["syntax error before: ","']'"]}]}]]
2016-12-16 13:17:55.267 [error] <0.2.0>@boss_load:compile_and_accumulate_errors:258 Compile Error, "src/test/functional/server_test.erl" -> [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_test.erl",[{10,erl_parse,["syntax error before: ","']'"]}]}]
2016-12-16 13:17:55.267 [error] <0.2.0>@boss_load:make_all_modules:88 Load Module Error test_modules : [[{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_test.erl",[{10,erl_parse,["syntax error before: ","']'"]}]}]]
2016-12-16 13:21:26.005 [error] <0.2.0>@boss_load:compile_and_accumulate_errors:258 Compile Error, "src/test/functional/server_test.erl" -> [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_test.erl",[{13,erl_parse,["syntax error before: ","'('"]}]}]
2016-12-16 13:21:26.005 [error] <0.2.0>@boss_load:make_all_modules:88 Load Module Error test_modules : [[{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_test.erl",[{13,erl_parse,["syntax error before: ","'('"]}]}]]
2016-12-16 13:21:26.886 [error] <0.2.0>@boss_load:compile_and_accumulate_errors:258 Compile Error, "src/test/functional/server_test.erl" -> [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_test.erl",[{13,erl_parse,["syntax error before: ","'('"]}]}]
2016-12-16 13:21:26.886 [error] <0.2.0>@boss_load:make_all_modules:88 Load Module Error test_modules : [[{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_test.erl",[{13,erl_parse,["syntax error before: ","'('"]}]}]]
2016-12-16 13:25:38.017 [error] <0.2.0>@boss_load:compile_and_accumulate_errors:258 Compile Error, "src/test/functional/server_test.erl" -> [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_test.erl",[{16,erl_parse,["syntax error before: ","':'"]}]}]
2016-12-16 13:25:38.017 [error] <0.2.0>@boss_load:make_all_modules:88 Load Module Error test_modules : [[{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_test.erl",[{16,erl_parse,["syntax error before: ","':'"]}]}]]
2016-12-16 13:25:39.230 [error] <0.2.0>@boss_load:compile_and_accumulate_errors:258 Compile Error, "src/test/functional/server_test.erl" -> [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_test.erl",[{16,erl_parse,["syntax error before: ","':'"]}]}]
2016-12-16 13:25:39.230 [error] <0.2.0>@boss_load:make_all_modules:88 Load Module Error test_modules : [[{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_test.erl",[{16,erl_parse,["syntax error before: ","':'"]}]}]]
2016-12-16 14:42:36.267 [error] <0.2.0>@boss_load:compile_and_accumulate_errors:258 Compile Error, "src/test/functional/server_tests.erl" -> [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_tests.erl",[{16,erl_parse,["syntax error before: ","':'"]}]}]
2016-12-16 14:42:36.267 [error] <0.2.0>@boss_load:make_all_modules:88 Load Module Error test_modules : [[{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_tests.erl",[{16,erl_parse,["syntax error before: ","':'"]}]}]]
2016-12-16 14:43:49.639 [error] <0.2.0>@boss_load:compile_and_accumulate_errors:258 Compile Error, "src/test/functional/server_tests.erl" -> [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_tests.erl",[{16,erl_parse,["syntax error before: ","':'"]}]}]
2016-12-16 14:43:49.639 [error] <0.2.0>@boss_load:make_all_modules:88 Load Module Error test_modules : [[{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_tests.erl",[{16,erl_parse,["syntax error before: ","':'"]}]}]]
2016-12-16 14:43:50.501 [error] <0.2.0>@boss_load:compile_and_accumulate_errors:258 Compile Error, "src/test/functional/server_tests.erl" -> [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_tests.erl",[{16,erl_parse,["syntax error before: ","':'"]}]}]
2016-12-16 14:43:50.501 [error] <0.2.0>@boss_load:make_all_modules:88 Load Module Error test_modules : [[{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_tests.erl",[{16,erl_parse,["syntax error before: ","':'"]}]}]]
2016-12-16 14:44:38.048 [error] <0.2.0>@boss_load:compile_and_accumulate_errors:261 Compile Error, "src/test/functional/server_tests.erl" -> [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_tests.erl",[{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr}]}]
2016-12-16 14:44:38.049 [error] <0.2.0>@boss_load:make_all_modules:88 Load Module Error test_modules : [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_tests.erl",[{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr}]}]
2016-12-16 14:44:38.960 [error] <0.2.0>@boss_load:compile_and_accumulate_errors:261 Compile Error, "src/test/functional/server_tests.erl" -> [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_tests.erl",[{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr}]}]
2016-12-16 14:44:38.960 [error] <0.2.0>@boss_load:make_all_modules:88 Load Module Error test_modules : [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_tests.erl",[{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr}]}]
2016-12-16 15:47:39.596 [error] <0.2.0>@boss_load:compile_and_accumulate_errors:261 Compile Error, "src/test/functional/server_tests.erl" -> [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_tests.erl",[{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr}]}]
2016-12-16 15:47:39.596 [error] <0.2.0>@boss_load:make_all_modules:88 Load Module Error test_modules : [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_tests.erl",[{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr}]}]
2016-12-16 15:47:40.820 [error] <0.2.0>@boss_load:compile_and_accumulate_errors:261 Compile Error, "src/test/functional/server_tests.erl" -> [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_tests.erl",[{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr}]}]
2016-12-16 15:47:40.820 [error] <0.2.0>@boss_load:make_all_modules:88 Load Module Error test_modules : [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_tests.erl",[{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr}]}]
2016-12-16 15:48:47.996 [error] <0.2.0>@boss_load:compile_and_accumulate_errors:261 Compile Error, "src/test/functional/server_tests.erl" -> [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_tests.erl",[{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr}]}]
2016-12-16 15:48:47.996 [error] <0.2.0>@boss_load:make_all_modules:88 Load Module Error test_modules : [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_tests.erl",[{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr}]}]
2016-12-16 15:48:49.065 [error] <0.2.0>@boss_load:compile_and_accumulate_errors:261 Compile Error, "src/test/functional/server_tests.erl" -> [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_tests.erl",[{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr}]}]
2016-12-16 15:48:49.065 [error] <0.2.0>@boss_load:make_all_modules:88 Load Module Error test_modules : [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_tests.erl",[{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr},{17,erl_lint,illegal_expr}]}]
2016-12-16 15:50:46.984 [error] <0.2.0>@boss_load:compile_and_accumulate_errors:261 Compile Error, "src/test/functional/server_tests.erl" -> [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_tests.erl",[{15,erl_lint,illegal_expr},{15,erl_lint,illegal_expr},{15,erl_lint,illegal_expr},{15,erl_lint,illegal_expr},{15,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr}]}]
2016-12-16 15:50:46.984 [error] <0.2.0>@boss_load:make_all_modules:88 Load Module Error test_modules : [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_tests.erl",[{15,erl_lint,illegal_expr},{15,erl_lint,illegal_expr},{15,erl_lint,illegal_expr},{15,erl_lint,illegal_expr},{15,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr}]}]
2016-12-16 15:50:48.186 [error] <0.2.0>@boss_load:compile_and_accumulate_errors:261 Compile Error, "src/test/functional/server_tests.erl" -> [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_tests.erl",[{15,erl_lint,illegal_expr},{15,erl_lint,illegal_expr},{15,erl_lint,illegal_expr},{15,erl_lint,illegal_expr},{15,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr}]}]
2016-12-16 15:50:48.186 [error] <0.2.0>@boss_load:make_all_modules:88 Load Module Error test_modules : [{"/Users/ebadism/Documents/flaskserver/currencyfair_server/src/test/functional/server_tests.erl",[{15,erl_lint,illegal_expr},{15,erl_lint,illegal_expr},{15,erl_lint,illegal_expr},{15,erl_lint,illegal_expr},{15,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr},{16,erl_lint,illegal_expr}]}]
